applications/mne_scan/plugins/eegosports/eegosports.cpp

```diff
@@ -148,7 +137,7 @@ void EEGoSports::init()
     m_bWriteDriverDebugToFile = false;
     m_bCheckImpedances = false;
 
-    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString(QCoreApplication::applicationDirPath() + "/resources/3DLayouts/standard_waveguard64_duke.elc")).toString();
+    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString(QCoreApplication::applicationDirPath() + "../resources/general/3DLayouts/standard_waveguard64_duke.elc")).toString();
```

Comment:
the resources folder should reside in the applications path. If not you have maybe forgot to add the to the plugins resources in the .pro file? You can take a look at how we deploy resources [here](https://mne-cpp.github.io/pages/development/ci_deployment.html#resource-handling).

Reply:
Ok, I see. It seems though that not all layouts from mne-cpp/resources/3DLayouts are copied to mne-cpp/bin/resources/3DLayouts. If you could point me to where this can be changed, I'll correct this.

Reply:
This can be changed/adde [here](https://github.com/johaenns/mne-cpp/blob/aec9d068776269da44acdc13fd257b64f6aae444/applications/mne_scan/plugins/eegosports/eegosports.pro#L110)

Reply:
Ah, I see. Overlooked that in the document you linked. Is the idea to only copy the one electrode layout that is hard-coded in the plugin to ./bin/resources/ or should also some alternatives that are given in ./resources/ be copied?

Reply:
Is there a way for the user to change the electrode Lay-out during runtime? If yes then we should add all Lay-out files to be copied.